src/lib/serviceRoutines/postUpdateContext.cpp

```diff
@@ -711,15 +711,7 @@ std::string postUpdateContext
     // one used by the calling postBatchUpdate() function at serviceRoutineV2 library
     if (fails == response.contextElementResponseVector.size())
     {
-      // If all CER result in error, then it isn't a partial update, but a regular NotFound
-      if (ciP->apiVersion == V1)
```

Comment:
This block already runs in a `if (ciP->apiVersion == V2)` so we don't need to re-check here.

NTC (just informative)